### ISSUE TITLE
0.71 uses black but not in requirements

### ISSUE THREAD

**Issue by aenima-x** [ikamensh/flynt#118]
In 0.71 you use black in src/flynt/pyproject_finder.py 
But its not a requirement so it crashes

```
$ pip install flynt
Collecting flynt
  Downloading flynt-0.71-py3-none-any.whl (28 kB)
Collecting tomli>=1.1.0
  Downloading tomli-2.0.0-py3-none-any.whl (12 kB)
Collecting astor
  Downloading astor-0.8.1-py2.py3-none-any.whl (27 kB)
Installing collected packages: tomli, astor, flynt
Successfully installed astor-0.8.1 flynt-0.71 tomli-2.0.0

$ flynt -df xxx tests
Traceback (most recent call last):
  File "/usr/local/bin/flynt", line 5, in <module>
    from flynt import main
  File "/usr/local/lib/python3.10/site-packages/flynt/__init__.py", line 7, in <module>
    from flynt.cli import main
  File "/usr/local/lib/python3.10/site-packages/flynt/cli.py", line 10, in <module>
    from flynt.pyproject_finder import find_pyproject_toml, parse_pyproject_toml
  File "/usr/local/lib/python3.10/site-packages/flynt/pyproject_finder.py", line 12, in <module>
    from black import err
ModuleNotFoundError: No module named 'black'
```

**Comment by ikamensh** [comment]
fixed now, v0.72 uploaded

**Comment by ikamensh** [comment]
thanks for the bug report btw :)
